pht_federated/aggregator/storage/db/engine.py

```diff
@@ -1,5 +1,11 @@
 from sqlmodel import create_engine
+from sqlalchemy.orm import sessionmaker
+
 
 ENGINE_URL = 'postgresql+psycopg2://postgres:postgres@localhost/postgres'
 
+ENGINE_URL = "postgresql+psycopg2://admin:admin@localhost:5442/pht_station"
+
 engine = create_engine(ENGINE_URL)
```

Comment:
make hardcoded connection optional and enable configuration via environment variable